src/app/frontend/logs/logstoolbar/logstoolbar.scss

```diff
@@ -14,6 +14,10 @@
 
 $logs-toolbar-color-black: #000;
 $logs-toolbar-color-white: #fff;
+$logs-toolbar-text-font-size: 17px;
```

Comment:
Can you use any of font sizes from https://github.com/angular/material/blob/HEAD/src/core/style/variables.scss and define the font in src/app/frontend/_variables.scss ?